packages/circuit-ui/components/Tooltip/Tooltip.tsx

```diff
@@ -114,37 +134,19 @@ const positionAndAlignStyles = ({
   theme,
   position = 'right',
   align = 'center',
-}) => css`
+}: StyleProps & TooltipProps) => css`
   ${getAlignmentStyles({ theme, position, align })};
   ${getPositionStyles({ theme, position })};
 `;
 
+type TooltipProps = { position?: Position; align?: Alignment };
```

Comment:
Conventionally, we use interfaces for component prop types so they can be extended. The props interface should also be exported alongside the component.

Reply:
Thank you.

Reply:
I see that you've exported from the component, thanks for making the change. It should be exported publicly from the package as well :)